source/includes/events/_auth.needs-resync.md.erb

```diff
@@ -0,0 +1,36 @@
+## Auth needs-resync
```

Comment:
This hyphen between "needs-resync" isn't needed, unless it must be included for code-related reasons.